### DISCUSSION TITLE
Add basic permission config button for public sharing to Public role similar to "app access"

### DISCUSSION THREAD

**Discussion by PeterEhses** [directus/directus#12705] [Archive]
When setting up shares, I had to configure the public role for some non-intuitive permissions (e.g. read directus rule, read directus permissions) as well as set up a Permission group for the shares. 
It'd be nice to have a "shares access" button on the Public permissions group just like one can enable "app access" on user groups. 
Ideally, this would configure Public and add a "share user" group with minimum permissions needed.

**Comment by rijkvanzanten** [comment]
> When setting up shares, I had to configure the public role for some non-intuitive permissions (e.g. read directus rule, read directus permissions) as well as set up a Permission group for the shares.

Could you elaborate on this? What do you mean by _setting up shares_? You shouldn't have to setup any special rules on the public role in order to create a share 🤔

**Reply by PeterEhses** [reply]
Never mind, I misunderstood the documentation at first. That, combined with the misconfiguration / potential bug in my database caused me some confusion. 
I've moved the discussion to Archive.
Feel free to lock / delete.

**Reply by rijkvanzanten** [reply]
cc @erondpowell

**Reply by erondpowell** [reply]
> Never mind, I misunderstood the documentation at first. That, combined with the misconfiguration / potential bug...

Which part of the docs were misunderstood? Is there anything you can think of to help make them clearer?
